upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -60,6 +60,9 @@ const starline = "**************************************************************
 // AlphaAllowGCE is a feature flag that gates GCE support while it is alpha
 var AlphaAllowGCE = featureflag.New("AlphaAllowGCE", featureflag.Bool(false))
 
+// AlphaAllowVsphere is a feature flag that gates vsphere support while it is alpha
+var AlphaAllowVsphere = featureflag.New("AlphaAllowVsphere", featureflag.Bool(false))
```

Comment:
We need to figure out where to put these.  We have them in a few different files, and it may be a good idea to put them just in one place